corehq/apps/app_manager/models.py

```diff
@@ -2736,6 +2736,14 @@ class AdvancedModule(ModuleBase):
     search_config = SchemaProperty(CaseSearch)
 
     @classmethod
+    def wrap(cls, data):
+        # lazy migration to accommodate search_config as empty list
+        # http://manage.dimagi.com/default.asp?231186
+        if data.get('search_config') == []:
```

Comment:
is it possible that this won't be true? as in it already has values in the array?